test/scripts/environment.yml

```diff
@@ -0,0 +1,11 @@
+channels:
+  - conda-forge
+  - defaults
+
+dependencies:
+  - python=3.10
```

Comment:
Latest stable python is 3.12. Since this is a new conda env anyway maybe start there? What do you think?

Also, it's standard across Sofar to use black to format python code. Add black to the dependencies and format the python files accordingly.

Reply:
Sounds good I will add that.